packages/instrumentation-node-cache/src/node-cache.ts

```diff
@@ -0,0 +1,132 @@
+import type * as NodeCache from 'node-cache';
```

Comment:
@blumamir I know the recommendation is to have to types as a dependency (and not devDep).  
But in this case, the types are part of the main lib, any recommendation here?